### ISSUE TITLE
TIFF tag 700 (XMP) does not show up as an existing tag in Directory.getTags() function

### ISSUE THREAD

**Issue by PaMcZee** [drewnoakes/metadata-extractor#320]
When retrieving the listing of tag values for a TIFF file that contains tag 700 (XMP Data), that tag number is not listed as an existing tag.   

I have attached a sample TIFF that contains tag 700. 

Here is a small sample where tag 700 is not listed.
      
``` 
File fi = new File("Arbito.tif");
Metadata metadata = ImageMetadataReader.readMetadata(fi);
for ( Directory directory : metadata.getDirectories()) 
{
	for ( Tag tag : directory.getTags())
		System.out.println(tag.getTagType()); // no tag 700 is outputted
}

```

I am aware that the existence of XMP data can be determined by the Directory type (XMPDirectory), however I feel it is confusing to not list TIFF tag 700 in the list of existing tags for a stock Directory.   

Also, the tag value that I see for the XMPDirectory is 65535, thus making this more confusing.  It's as though the 700 goes down the memory hole and no way to retrieve it or even know about it, even in a roundabout way,    Looking at the source code, it seems that this is the case (the source detects type 700, but then doesn't retain it, store it, or anything else that would hint that it is even there).   

The internal loop that gets the tiff tags, **TiffReader.processIfd()**, even shows that the number of tags retrieved is "n", but only indicates back to the client that "n - 1"  tags exist (the one missing is 700).  This seems wrong to me.

So in short, the code above should output all tags (and it does, except for tag 700, which it should).  

If I am doing something wrong or if there is a way to simply list tag 700 that I'm not aware of, then I will be glad to be corrected on this matter.
  
Thanks.

[Arbitro.zip](https://github.com/drewnoakes/metadata-extractor/files/1683307/Arbitro.zip)
